libopae/plugins/xfpga/feature_plugins/feature_token_list.c

```diff
@@ -74,9 +76,10 @@ struct _fpga_feature_token *feature_token_add(uint32_t type, uint32_t mmio_num,
 		if ((uuid_compare(guid, tmp->feature_guid)) == 0) {
 			err = pthread_mutex_unlock(&ftoken_lock);
 			if (err) {
-				FPGA_ERR("pthread_mutex_unlock() failed: %S",
+				FPGA_ERR("pthread_mutex_unlock() failed: %s",
 					 strerror(err));
 			}
+			printf("feature_token_add found token in the list, return immedaitely\n");
```

Comment:
Remove debug code, or convert to an appropriate FPGA_MSG, etc.
s/immedaitely/immediately/

Reply:
Removed.